muspinsim/constants.py

```diff
@@ -14,11 +14,14 @@
 MU_TAU = 2.19703  # Muon decay rate (10^-6 s)
 # EFG to MHz constant for Quadrupole couplings
 # (the total quadrupole coupling in MHz is QCONST*Q*Vzz)
+# The factor 1e-36 comes from 1e-6 * 10e-28 * 1e-3
+# in order this converts to MHz, from barn to m^2 and the last factor
+# accounts for the soprano returning quadrupole moments in millibarn
```

Comment:
Thanks for making this more explicit. To be honest I still don't know why it's `10e-28` rather than `1e-28` given that a `1 barn = 100 fm**2 = 10**2 * (10**-15)**2 m**2 = 10**-28 m**2`, but documenting it to prevent futrure confusion is the important thing.

Reply:
@patrick-austin Yes, 1 barn = 10**-28 m**2 = 10e-28 = 1e-29. 1e-28 is instead equivalent to 10**-27 m**2. For example 10e2 = 1000, 1e2 = 100.

Reply:
`10**-28 = 1 * 10**-28 = 1e-28`, the `e` or `E` is shorthand for `* 10**`? I absolutely agree that `10e2 is 1000`, but not that `10**-28 = 10e-28`:
```
$ python
Python 3.10.3 (tags/v3.10.3:a342a49, Mar 16 2022, 13:07:40) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> 10**-28
1e-28
>>> 10e-28
1e-27
```

Reply:
Ah, ok I see now, I need to double check that before merging.

Reply:
@patrick-austin Yep turns out it was correct originally. Because UNDI required the quadrupole moment to be specified explicitly I had given it values of -52*1e-32, which was incorrect and should have been -52*1e-31 because I made the same mistake. It also turns out once this is rectified the value of hbar was also incorrect and it was previously correct. The extra 2pi is accounted for elsewhere during the evolution itself. Reverting EFG_2_MHZ to its original value also recreates a much more reasonable match to the paper result as well so I am confident it was right all along it was just my misuse of UNDI causing the issues. Thanks a lot for spotting this.